src/controllers/auth.controller.js

```diff
@@ -0,0 +1,89 @@
+require('dotenv').config();
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const path = require('path');
+const db = require(path.join(__dirname, '../db/config'));
+const queries = require(path.join(__dirname, '../db/queries'));
+const saltRounds = 10;
+
+const authenticate = async (user) => {
+    const expiresIn = '2 days';
+    const accessToken = jwt.sign(
+        { id: user.id },
+        process.env.SECRET,
+        { expiresIn: expiresIn }
+    );
+
+    return {
+        user: user,
+        access_token: accessToken,
+        expires_in: expiresIn
+    }
+}
+
+// const authorize = async (password, hash) => {
```

Comment:
Note `password` is the user-sent password and `hash` is the hashed password retrieved from the db (referred to respectively as `userSentPassword` and `password` in the `login` func below).